documentation/specs/proposed/BuildCheck.md

```diff
@@ -128,9 +130,20 @@ We might as well consider specifying custom analyzers on a command line (as a no
 
 There will be 3 mechanisms of configuring the analyzers and rules:
 * The default configuration declared by the analyzers themselves ([more details on implementation](#rules-declaration))
-* [Sdk Analysis Level property](https://github.com/dotnet/designs/blob/main/proposed/sdk-analysis-level.md) – mostly for the inbox analyzers
+* The TFM of the project and the [Sdk Analysis Level property](https://github.com/dotnet/designs/blob/main/proposed/sdk-analysis-level.md) – mostly for the inbox analyzers
 * `.editorconfig` file
 
+We will also consider respecting `SdkAnalysisLevel` to override the per-TFM defaults. Additionally we may use the existing `RunAnalyzers` "master switch" property to make it possible to disable everything whole-sale. This would be used in scenarios like F5 in VS.
```

Comment:
We should consider a different property name than `RunAnalyzers` for a couple reasons:
* We use the term BuildChecks now
* This might confuse users thinking that it also applies to Roslyn Analyzers

Maybe something like `RunMSBuildAnalyzers` or `RunMSBuildChecks`?

Reply:
Thank you, I've changed it to `RunMSBuildChecks`.